### PR TITLE
Feature users profiles

### PR DESCRIPTION
**Pull request by medBelaid**
- update design menu
- creation controller profile

### CODE REVIEW

**Comment by medBelaid** [comment]
@DoniaMroua @SamiDghim @melas123 @achr3f plz take a look at this code

File: app/views/profile/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+%h1 Profile#index
```
Comment (SamiDghim) [inline app/views/profile/index.html.haml:1]: please remove auto-generated files

Comment (melas123) [inline app/views/profile/index.html.haml:1]: delete unused files ..

File: spec/helpers/profile_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (melas123) [inline spec/helpers/profile_helper_spec.rb]: this is also  unecessary

Comment (medBelaid) [inline spec/helpers/profile_helper_spec.rb]: ok

File: app/controllers/profile_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class ProfileController < ApplicationController
+  before_filter :authenticate_user!, only: [ :create, :update, :destroy ]
+  respond_to :json
+  def index
+    #return recipes's current_user.
```
Comment (SamiDghim) [inline app/controllers/profile_controller.rb:5]: before you return recipe for the current user you must test if user signed in : 
 if user_signed_in?
  #return user recipes
, and plz use named scope

Comment (medBelaid) [inline app/controllers/profile_controller.rb:5]: we have before_filter : authentificate_user!

2016-01-25 11:08 GMT+01:00 Sami notifications@github.com:

Comment (SamiDghim) [inline app/controllers/profile_controller.rb:5]: yes, this is only for create , update and destroy actions

File: app/assets/javascripts/templates/auth/_register.html.haml [hunk 4]
```diff
@@ -1,12 +1,16 @@
 .page-header
-  %h1 Register
+  %h1 Inscription
 %form.form-horizontal{ ng_submit: "register()" }
   .form-group
+    %label.control-label.col-sm-6 Nom:
```
Comment (SamiDghim) [inline app/assets/javascripts/templates/auth/_register.html.haml:5]: Use the English language in your view

File: app/assets/javascripts/templates/auth/_register.html.haml [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
     %label.control-label.col-sm-6 Email:
     .col-sm-6
-      %input.form-control{ ng_model: "user.email", placeholder: "Email", type: "email" }
+      %input.form-control{ ng_model: "user.email", placeholder: "Email ...", type: "email" }
   .form-group
     %label.control-label.col-sm-6 Password:
     .col-sm-6
-      %input.form-control{ ng_model: "user.password", placeholder: "Password", type: "password" }
+      %input.form-control{ ng_model: "user.password", placeholder: "Password ...", type: "password" }
   %input.btn.btn-primary.pull-right{ type: "submit", value: "Register" }
```
Comment (thameursaadi) [inline app/assets/javascripts/templates/auth/_register.html.haml]: choose one language (nom..password..register)

File: app/assets/javascripts/templates/auth/_register.html.haml (hunk 4 above)
Comment (medBelaid) [inline app/assets/javascripts/templates/auth/_register.html.haml:5]: our client interfaces are in french

File: app/assets/javascripts/templates/auth/_register.html.haml (hunk 5 above)
Comment (medBelaid) [inline app/assets/javascripts/templates/auth/_register.html.haml]: done, french

2016-01-25 11:26 GMT+01:00 thameursaadi notifications@github.com:

**Pull request merged**